drivers/delta-hid.c

```diff
@@ -0,0 +1,188 @@
+#include "main.h"	/* for getval() */
+#include "usb-common.h"
+
+#define DELTA_HID_VERSION	"Delta HID 0.5"
+
+/* Delta Vendor ID */
+#define DELTA_VENDORID	0x05dd
+
+/* USB IDs device table */
+static usb_device_id_t delta_usb_device_table[] = {
+	/* Delta RT Series, Single Phase, 1/2/3 kVA */
+	{ USB_DEVICE(DELTA_VENDORID, 0x041b), NULL },
+
+	/* Terminating entry */
+	{ -1, -1, NULL }
+};
+
+
+/* --------------------------------------------------------------- */
+/*      Vendor-specific usage table */
+/* --------------------------------------------------------------- */
+
+/* DELTA usage table */
+static usage_lkp_t delta_usage_lkp[] = {
+	{ "DELTA1",	0x00000000 },
+	{ "DELTA2",	0xff000055 },
+	{ "DELTA3",	0xffff0010 },
+	{ "DELTA4",	0xffff0056 },
+	{ "DELTA5",	0xffff0057 },
+	{ "DELTA6",	0xffff0058 },
+	{ "DELTA7",	0xffff0059 },
+	{ "DELTA8",	0xffff005a },
+	{ "DELTA9",	0xffff0060 },
+	{ "DELTA10",	0xffff0061 },
+	{ "DELTA11",	0xffff0062 },
+	{ "DELTA12",	0xffff0063 },
+	{ "DELTA13",	0xffff0064 },
+	{ "DELTA14",	0xffff0065 },
+	{ "DELTA15",	0xffff0066 },
+	{ "DELTA16",	0xffff0067 },
+	{ "DELTA17",	0xffff0068 },
+	{ "DELTA18",	0xffff0075 },
+	{ "DELTA19",	0xffff0076 },
+	{ "DELTA20",	0xffff007c },
+	{ "DELTA21",	0xffff007d },
+	{ "DELTA22",	0xffff0081 },
+	{ "DELTA23",	0xffff0091 },
+	{ "DELTA24",	0xffff0092 },
+	{ "DELTA25",	0xffff0093 },
+	{ "DELTA26",	0xffff0094 },
+	{ "DELTA27",	0xffff0095 },
+	{ "DELTA28",	0xffff0096 },
+	{ "DELTA29",	0xffff0097 },
+	{ "DELTA30",	0xffff0098 },
+	{ "DELTA31",	0xffff0099 },
+	{ "DELTA32",	0xffff009a },
+	{ "DELTA33",	0xffff009b },
+	{ "DELTA34",	0xffff009c },
+	{  NULL, 0 }
+};
+
+static usage_tables_t delta_utab[] = {
+	delta_usage_lkp,
+	hid_usage_lkp,
+	NULL,
+};
+
+/* --------------------------------------------------------------- */
+/* HID2NUT lookup table                                            */
+/* --------------------------------------------------------------- */
+
+static hid_info_t delta_hid2nut[] = {
+
+  { "battery.voltage.nominal", 0, 0, "UPS.BatterySystem.Battery.ConfigVoltage", NULL, "%.0f", HU_FLAG_STATIC, NULL },
+  { "battery.voltage", 0, 0, "UPS.BatterySystem.Battery.Voltage", NULL, "%.1f", HU_FLAG_QUICK_POLL, NULL },
+  { "battery.temperature", 0, 0, "UPS.BatterySystem.Temperature", NULL, "%s", HU_FLAG_QUICK_POLL, kelvin_celsius_conversion },
+  { "test.battery.start.quick", 0, 0, "UPS.BatterySystem.Test", NULL, "1", HU_TYPE_CMD, NULL },
+  { "ups.power.nominal", 0, 0, "UPS.Flow.ConfigApparentPower", NULL, "%.0f", HU_FLAG_STATIC, NULL },
+  { "output.frequency.nominal", 0, 0, "UPS.Flow.ConfigFrequency", NULL, "%.0f", HU_FLAG_SEMI_STATIC, NULL },
+  { "output.voltage.nominal", 0, 0, "UPS.Flow.ConfigVoltage", NULL, "%.0f", HU_FLAG_SEMI_STATIC, NULL },
+  { "ups.realpower", 0, 0, "UPS.OutletSystem.Outlet.ActivePower", NULL, "%.1f", HU_FLAG_QUICK_POLL, NULL },
+  { "ups.delay.reboot", 0, 0, "UPS.OutletSystem.Outlet.DelayBeforeReboot", NULL, "%.0f", HU_FLAG_SEMI_STATIC, NULL },
```

Comment:
these 3 delays are configs & RW, and used (duplicated) later to declare the commandss.
See 
https://github.com/networkupstools/nut/blob/master/drivers/mge-hid.c#L1188
https://github.com/networkupstools/nut/blob/master/drivers/mge-hid.c#L1394
note that you just need to declare the "delay"  version of the command, the instant one (delay = 0) will be automatically added